### ISSUE TITLE
models magically loading and not loading from time to time

### ISSUE THREAD

**Issue by sebdamememan** [edunad/qube#7]
this has become a recurring problem and I really dont see an answer to it, whenever I have an obj loaded in game, or when I get the opportunity to load one in, sometimes i can load it completely normal. then I open up the game again and suddenly "poof" failed to find 'Content-Range' header, I say "weird flex", open the game again and then "poof" object back to normal. any good ideas onto WHY it may be doing this from time to time?

**Comment by edunad** [comment]
what url host is the model using?

**Comment by sebdamememan** [comment]
The model itself is being hosted on google drive, should i try changing it to something else?

**Comment by sebdamememan** [comment]
The model itself , as well as its texture is being hosted on google drive, here:

Model:https://drive.google.com/file/d/1W_QBEGzxWjhHaYdNNhg6MEz658w_t5Dg/view?usp=drivesdk

Material:https://drive.google.com/file/d/1ZUzVCI2p-xbi-WE1ep5h0G-6dl67bKSU/view?usp=drivesdk

If for some reason i shouldnt be using google drive, or in the case that any other services like dropbox work better pls do tell

**Comment by edunad** [comment]
Seems to be google driving limiting the requests, can you try dropbox or pastebin?

I've added `prop_mesh_ignoreContentRange` to disable content range checking, this will make users / server download unlimited size files, so it's not really a good idea to disable it.

#8

**Comment by sebdamememan** [comment]
will do and report back on the results

**Comment by sebdamememan** [comment]
> Seems to be google driving limiting the requests, can you try dropbox or pastebin?
> 
> I've added `prop_mesh_ignoreContentRange` to disable content range checking, this will make users / server download unlimited size files, so it's not really a good idea to disable it.
> 
> #8

sadly, changing the link to drop box nor paste bin made it download constantly, it keeps on doing it one time, and then failing the other, also on another note, for some reason the in game console is not recognizing prop_mesh_ignoreContentRange as a valid command to use on multiplayer servers started on the single player menu

**Comment by edunad** [comment]
did you update the addon? i pushed the update to workshop yesterday

**Comment by sebdamememan** [comment]
100% sure

this is a screenshot of a multiplayer local server game started on the single player tab

![Captura de pantalla 2023-09-25 005737](https://github.com/edunad/prop_mesh/assets/145504829/f0834fdb-2b3a-4026-95c4-2c86ba6c2839)

and this is a pic of the same command being recognized by a server console from a server on the multiplayer tab

![Captura de pantalla 2023-09-25 010130](https://github.com/edunad/prop_mesh/assets/145504829/232e8271-5a05-4abf-a49b-ad2ae1b4774f)

**Comment by edunad** [comment]
> 100% sure
> 
> this is a screenshot of a multiplayer local server game started on the single player tab
> 
> ![Captura de pantalla 2023-09-25 005737](https://user-images.githubusercontent.com/145504829/270225086-f0834fdb-2b3a-4026-95c4-2c86ba6c2839.png)
> 
> and this is a pic of the same command being recognized by a server console from a server on the multiplayer tab
> 
> ![Captura de pantalla 2023-09-25 010130](https://user-images.githubusercontent.com/145504829/270225202-232e8271-5a05-4abf-a49b-ad2ae1b4774f.png)

On singleplayer did you at least select local server and a min of players? The addon does not work on pure singleplayer

**Comment by sebdamememan** [comment]
Yes, local server and a minimum of 4 players was selected, I am aware that the addon doesnt work on pure single player games, even with all of that the console aint loading up the command

**Comment by sebdamememan** [comment]
as a note, i have tried to uninstall and install the addon, yet the command in a local server game is not being recognized by the console
